src/server/controllers/user.js

```diff
@@ -2,33 +2,43 @@ const bcrypt = require('bcrypt');
 const jwt = require('jsonwebtoken');
 const { PrismaClient } = require('@prisma/client');
 const prisma = new PrismaClient();
+const saltRounds = 10;
 
 const jwtSecret = 'mysecret';
 
 const register = async (req, res) => {
     const { username, password } = req.body;
-
-    const createdUser = null;
-
-    res.json({ data: createdUser });
+    const hashedPassword = await bcrypt.hash(password, saltRounds);
```

Comment:
I like how you used a variable for the salt rounds. I completely forgot to do that.